Bingha/Bingha/Views/Home/Complete/CompleteViewController.swift

```diff
@@ -12,31 +12,58 @@ class CompleteViewController: UIViewController {
     //스토리보드 연결
     
     //모달 닫기 버튼
-    @IBOutlet weak var exitButton: UIImageView!
+    @IBOutlet weak var exitModalButton: UIButton!
     //상세 데이터 백그라운드 뷰
-    @IBOutlet weak var dataBackground: UIView!
+    @IBOutlet weak var dataView: UIView!
+    //빙하 레벨업 알림 뷰
+    @IBOutlet weak var icebergLevelAlertView: UIView!
     
     //~~동안 산소로 바꾸는 양
-    @IBOutlet weak var oxygenChangeAmount: UILabel!
+    @IBOutlet weak var changeToOxyLabel: UILabel!
     //탄소 배출 저감량
-    @IBOutlet weak var reducedCarbon: UILabel!
+    @IBOutlet weak var reducedCarbonLabel: UILabel!
     //오늘 총 저감량
-    @IBOutlet weak var todayReducedCarbon: UILabel!
+    @IBOutlet weak var todayReducedCarbonLabel: UILabel!
     //이동거리
-    @IBOutlet weak var moveDistance: UILabel!
+    @IBOutlet weak var moveDistanceLabel: UILabel!
     //소요시간
-    @IBOutlet weak var timeDuration: UILabel!
+    @IBOutlet weak var timeDurationLabel: UILabel!
     
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        //View 커스텀
+        viewCornerAndShadowCustom(viewName: icebergLevelAlertView) //빙하 레벨업
+        viewCornerAndShadowCustom(viewName: dataView) //상세 데이터
+        exitButtonCustom()
+        
+        //Label 폰트 커스텀
+        
+        //데이터 연결
+        
+        
+        
+    }
+    
+    //View 커스텀 함수
+    func viewCornerAndShadowCustom(viewName: UIView) {
```

Comment:
```suggestion
    func viewCornerAndShadowCustom(view: UIView) {
```
Argument Label에는 대입하는 객체타입 자체를 명시하는 이름이 좋습니다. 함수 내부에서는 해당 객체를 대신해서 사용하는 parameter name으로 사용하기 때문에 코드를 작성하는데 이해하기 쉬워집니다!

Reply:
오 그렇군요. 참고해서 코드 작성해보겠습니다!